README.md

```diff
@@ -427,8 +428,8 @@ spec:
     rotationFrequency: 600h
  ```
 
-Once the `ServiceBinding` is rotated, the original binding will be updated with new credentials. 
-A new `ServiceBinding` named '<original-binding-name>-<guid>' will be created as a backup of the old credentials, which will be deleted according to the configured TTL. <br>
+Once the `ServiceBinding` is rotated, the original `ServiceBinding` will be updated with a new binding, the old binding will be saved in a new `ServiceBinding` 
+named '<original-binding-name>-<guid>', which will be deleted according to the configured TTL. <br>
 To see when the service binding was last rotated, refer to the `status.lastCredentialsRotationTime`. <br>
```

Comment:
What about the secret?
need to explain that the original secret is updated with the new set of credentials. and that the old credentials are temporarily kept in a secret named <backup secret name>.

Reply:
We mention secret in the section about Bindings. Perhaps we can add a reference here for users who did not read about it.